src/repository/mongo/CredentialRepository.ts

```diff
@@ -85,6 +88,19 @@ export class CredentialRepository implements CredentialInterface {
     return {}
   }
 
+  public async _tokens(access_key: string): Promise<Credential["tokens"]> {
```

Comment:
This function seems to map `access_key → tokens[]`. Could we add a function to map `token → access_key` to look up the "owner credential" of a token?

Reply:
I thought we would want to check it anyway just to be sure it wasn't already revoked?

Reply:
with the credential access key we retrieve the credential to be sure that it wasn't already revoked.
The origin is not encoded in the token, instead that is retrieved from the credential itself.

Reply:
Perfect, makes sense. Thanks!